source/Engine.cpp

```diff
@@ -1202,7 +1202,7 @@ void Engine::BreakTargeting(const Government *gov)
 
 
 
-void Engine::EnterSystem()
+void Engine::EnterSystem(System &previousSystem)
```

Comment:
`const`?

Reply:
this part is temporary it wont stay, just wrote it to get this previous system but it'll be provided by playerInfo

that said I need it non const anyway later on so I don't think it'd be worth it

Reply:
https://github.com/endless-sky/endless-sky/pull/7974 this will provide the means to get it
but I'll have to const cast it anyway